dissect/target/loaders/mqtt.py

```diff
@@ -423,16 +425,149 @@ def connect(self) -> None:
         self.mqtt_client.loop_start()
 
 
-@arg("--mqtt-peers", type=int, dest="peers", help="minimum number of peers to await for first alias")
-@arg("--mqtt-case", dest="case", help="case name (broker will determine if you are allowed to access this data)")
-@arg("--mqtt-port", type=int, dest="port", help="broker connection port")
-@arg("--mqtt-broker", dest="broker", help="broker ip-address")
-@arg("--mqtt-key", dest="key", help="private key file")
-@arg("--mqtt-crt", dest="crt", help="client certificate file")
-@arg("--mqtt-ca", dest="ca", help="certificate authority file")
+def strictly_positive(value: str) -> int:
+    """
+    Validates that the provided value is a strictly positive integer.
+
+    This function is intended to be used as a type for argparse arguments.
+
+    Args:
+        value (str): The value to validate.
+
+    Returns:
+        int: The validated integer value.
+
+    Raises:
+        argparse.ArgumentTypeError: If the value is not a strictly positive integer.
+    """
+    try:
+        strictly_positive_value = int(value)
+        if strictly_positive_value < 1:
+            raise argparse.ArgumentTypeError("Value must be strictly positive.")
```

Comment:
```suggestion
            raise argparse.ArgumentTypeError("Value must be larger then or equal to 1.")
```
I'd phrase it like this.